static_frame/core/fill_value_auto.py

```diff
@@ -6,34 +6,83 @@
 from static_frame.core.util import NAT
 from static_frame.core.util import NAT_TD64
 
+FILL_VALUE_AUTO_DEFAULT = object()
```

Comment:
I notice this name is the same as `container_util.FILL_VALUE_AUTO_DEFAULT`, which could be confusing. Maybe call this `FILL_VALUE_UNSET` or something?

Reply:
I like `FILL_VALUE_UNSET`! I read the code before reading this comment and got confused.

Reply:
Great point; thanks!